npm/test/clientfilemetataupload.test.ts

```diff
@@ -195,6 +195,30 @@ test("createMetadataInputBatches generates batches of the defined size", () => {
   expect(result[1]).toEqual([input4, input5])
 })
 
+test("createMetadataInputsAndFileMap returns metadata inputs and file map", () => {
+  const metadata: IFileMetadata[] = [mockMetadata1, mockMetadata2]
+  const client = new GraphqlClient("https://test.im", mockKeycloakInstance)
```

Comment:
I think this is copied from elsewhere, but we should use example.com when we need a realistic but fake URL. It's a reserved domain which is designed to be used in examples, whereas test.im is some random person's real website. 🙂

Reply:
Cheers for fixing the existing ones. 👍